src/forwarder.ts

```diff
@@ -138,6 +140,16 @@ export async function addStepFunctionLogGroup(aws: Aws, resources: any, stepFunc
   };
 }
 
+export function addDdSlsPluginTag(stepFunction: any) {
+  if (stepFunction === undefined || stepFunction === null) {
+    return;
+  }
+  if (!stepFunction?.Properties.hasOwnProperty("Tags")) {
```

Comment:
should it be "!stepFunction?.Properties.hasOwnProperty("Tags")" or "!stepFunction.Properties?.hasOwnProperty("Tags")"    I think line 144 has done the null check on object stepFunction.

Reply:
Thanks for the comment. I have changed the way of the implementation.